mirage/fixtures/members.js

```diff
@@ -1,6 +1,6 @@
 export default [
   { id: 1, username: 'citizen4' },
-  { id: 4, username: 'lana' },
+  { id: 4, username: 'lana', progresses: [{ chapter_id: 102, sequence_num: 5 }, { chapter_id: 105, sequence_num: 999 }] },
```

Comment:
There needs to be a way to switch between members so that different member data returning from mock API can be tested.